pagopa/pn-frontend
# PN-4608-accepted-status-i18n-tooltip-description-exchanged

## Short description
Exchanged the i18n texts for tooltip and description for the ACCEPTED status of a notification.

## List of changes proposed in this pull request
- Just change the `notifiche.json` files in pa-webapp / pf-webapp / pg-webapp, exchanging the values of `accepted-tooltip`  and `accepted-description`.

## How to test
Enter a notification detail, check the description. 
Then search for a notification which current status is ACCEPTED (example IUN QEUK-UDEP-TUKU-202303-N-1, comune di Palermo - Ettore Fieramosca) and check the tooltip.
